doc/man/goto-inspect.1

```diff
@@ -0,0 +1,33 @@
+.TH GOTO-INSPECT "2" "May 2023" "goto-inspect-5.81.0" "User Commands"
```

Comment:
Probably not worth pinning a date/version here.

Reply:
[This reference](https://man7.org/linux/man-pages/man7/man-pages.7.html) leads me to believe that the date is necessary in the title line.

It does look like the date we have though doesn't align with the proposed format in the reference I linked. I'm not sure how the tools behave if I leave it with an empty date, but it seems to be diverging from the expected format too much IMO.